t/unit/test_schedulers.py

```diff
@@ -1663,31 +1678,6 @@ def setup_method(self):
     def teardown_method(self):
         patch.stopall()
 
-    @patch("django_celery_beat.schedulers.aware_now")
```

Comment:
can we add more tests which also improve the test coverage? and there is no regression introduced?